common/src/main/java/com/wynntils/screens/base/widgets/SortOrderWidget.java

```diff
@@ -73,4 +75,19 @@ public void renderWidget(PoseStack poseStack, int mouseX, int mouseY, float part
                     sortTexture.height());
         }
     }
+
+    @Override
+    public List<Component> getTooltipLines() {
```

Comment:
I did not even realize this was duplicated across the PR...! :-)

So fixing this was certainly nice.

But what I meant was should we have something like
  `Component.translatable("screens.wynntils.wynntilsContentBook.sort." + sortableContentScreen.getContentSortOrder().name.toLowerCase() +".name")` instead... :)

I am not 100% sure, but I don't like the repetition. 3 copies is just about my personal rule of thumb when you are allowed to copy instead of generalize, but it always leaves a bad taste to let such instances pass.

You don't have to do this, but at least I'd like to hear an argument of why not. :)

Reply:
Fixed it.